includes/Checker/Abstract_Check_Runner.php

```diff
@@ -398,6 +452,21 @@ final public function get_include_experimental() {
 		return $this->get_include_experimental_param();
 	}
 
+	/**
+	 * Returns an array of categories for filtering the checks.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return array An array of categories.
+	 */
+	final public function get_categories() {
```

Comment:
This method should be protected, there's no public usage of it.

I missed that in the `get_include_experimental()` method above, could you update that to become protected as well?
```suggestion
	final protected function get_categories() {
```